README.md

```diff
@@ -46,9 +46,9 @@ settings = MySettings(
 )
 ```
 
-### Параметры
-- ***modules***: Список импортируемых python файлов с переменными. Дефолтное значение ```None```. 
-- ***prefix***: Префикс, с которым берутся переменные окружения. Дефолтное значение ```None```.
+### Parameters
+- ***modules***: List of Python modules with variables to import. Default ```None```. 
+- ***prefix***: The prefix with which the environment variables are taken. Default - ```None```.
 
     _settings.py_
     ```python
```

Comment:
[MD031 - Fenced code blocks should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.13.0/doc/Rules.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines)

```suggestion

    ```python
```